pkgs/development/tools/database/prisma-engines/default.nix

```diff
@@ -33,21 +32,23 @@ rustPlatform.buildRustPackage rec {
       "cuid-1.3.2" = "sha256-qBu1k/dJiA6rWBwk4nOOqouIneD9h2TTBT8tvs0TDfA=";
       "graphql-parser-0.3.0" = "sha256-0ZAsj2mW6fCLhwTETucjbu4rPNzfbNiHu2wVTBlTNe4=";
       "mysql_async-0.31.3" = "sha256-2wOupQ/LFV9pUifqBLwTvA0tySv+XWbxHiqs7iTzvvg=";
-      "postgres-native-tls-0.5.0" = "sha256-UYPsxhCkXXWk8yPbqjNS0illwjS5mVm3Z/jFwpVwqfw=";
+      "postgres-native-tls-0.5.0" = "sha256-4CftieImsG2mBqpoJFfyq0R2yd2EyQX4oddAwyXMDZc=";
+      "mongodb-3.0.0" = "sha256-1WQgY0zSZhFjt1nrLYTUBrpqBxpCCgKRSeGJLtkE6pw=";
     };
   };
 
-  nativeBuildInputs = [ pkg-config git ];
+  nativeBuildInputs = [ pkg-config ];
 
   buildInputs = [
     openssl
     protobuf
   ] ++ lib.optionals stdenv.hostPlatform.isDarwin [ Security ];
 
-  # FIXME: Workaround Rust 1.80 support by updating time to 0.3.36
-  # https://github.com/prisma/prisma-engines/issues/4989
+  # FIXME: fix this upstream and remove this patch with the next version update.
```

Comment:
Upstream uses an older compiler version which doesn't produce the warning (turned into an error by `#[deny(warnings)]`), temporarily just silence the warning in this file.